File: February 10, 2026.js
function getRelativeResults(results) {

  let [hour, minute, second] = results[0].split(":").map(Number);
  let winnerTime = hour * 3600 + minute * 60 + second;
  let resultsBehind = ["0"];


  for (let i = 1; i < results.length; i++) {

    [hour, minute, second] = results[i].split(":").map(Number);
    let totalTime = hour * 3600 + minute * 60 + second;
    let timeBehind = totalTime - winnerTime;
    let hourBehindAux = Math.floor(timeBehind / 3600);
    let hourBehind = String(Math.floor(timeBehind / 3600)).padStart(2, "0");
    let minuteBehind = Math.floor((timeBehind % 3600) / 60);
    let secondBehind = String(timeBehind % 60).padStart(2, "0");

    if (hourBehindAux > 0) {
      resultsBehind.push(`+${hourBehind}:${minuteBehind}:${secondBehind}`);
    }

    resultsBehind.push(`+${minuteBehind}:${secondBehind}`)

  }

  return resultsBehind;
}

File: February 10, 2026.py
def get_relative_results(results):

    hour, minute, second = map(int, results[0].split(":"))
    winner_time = hour * 3600 + minute * 60 + second

    results_behind = ["0"]

    for i in range(1, len(results)):

        hour, minute, second = map(int, results[i].split(":"))
        total_time = hour * 3600 + minute * 60 + second

        time_behind = total_time - winner_time

        hour_behind_aux = time_behind // 3600
        hour_behind = str(time_behind // 3600).zfill(2)
        minute_behind = (time_behind % 3600) // 60
        second_behind = str(time_behind % 60).zfill(2)

        if hour_behind_aux > 0:
            results_behind.append(f"+{hour_behind}:{minute_behind}:{second_behind}")

        results_behind.append(f"+{minute_behind}:{second_behind}")

    return results_behind